src/ompl/geometric/planners/rrt/RRTstar.h

```diff
@@ -215,6 +216,24 @@ namespace ompl
                 return useInformedSampling_;
             }
 
+            /** \ Use intelligent sampling to bias samples towards the beacons */
+            void setIntelligentSampling(bool intelligentSampling);
```

Comment:
I don't like this as a name for the option. It does not explain what it is doing. How about `setSolutionBiasedSampling` or `setBeaconSampling` or something that is a little more descriptive.